packages/patternfly-4/react-integration/README.md

```diff
@@ -1,7 +1,33 @@
-# react-integration
+# Patternfly 4 React Integration Testing
+
+## Overview
+Here you will find a demo application written in typescript, along with cypress integration testing.  This is used to allow developers to verify their code against a working react app that is running their latest PF4 react changes.
+
+New componets should be written in typescript and added to the demo app.  Contributors should write cypress tests to test the functionality of their components.  Contributors should also test all of the user facing interfaces and components that they are creating.
+
+## Building your react component
+After completing the development of their component a contributor should perform a build so they can start writing integration tests.
 
 1. Make your changes in packages/patternfly-4/react-core
-2. `yarn build` in packages/patternfly-4/react-core
-3. `yarn integration:setup` in packages/patternfly-4/react-integration
-4. Update the demo in packages/patternfly-4/react-integration/demo-app-ts/src/App.tsx
-5. `yarn build:demo` in packages/patternfly-4/react-integration/demo-app-ts
+2. Type `yarn build` to build your component.
+
+## Running the demo app
```

Comment:
Running the demo app
Run the following command to launch the demo app:

`yarn start demo:app`

Once the demo app is launched, all saved changes you make to the code should be applied automatically.

Reply:
Done.